tests/sklearn/test_sklearn_autolog.py

```diff
@@ -206,37 +206,52 @@ def test_classifier():
     # use RandomForestClassifier that has method [predict_proba], so that we can test
     # logging of (1) log_loss and (2) roc_auc_score.
     model = sklearn.ensemble.RandomForestClassifier(max_depth=2, random_state=0, n_estimators=10)
-    X, y_true = get_iris()
+
+    # use binary datasets to cover the test for roc curve & precision recall curve
+    X, y_true = sklearn.datasets.load_breast_cancer(return_X_y=True)
 
     with mlflow.start_run() as run:
         model = fit_model(model, X, y_true, "fit")
 
     y_pred = model.predict(X)
     y_pred_prob = model.predict_proba(X)
+    # For bonary classification, y_score only accepts the probability of greater label
+    y_pred_prob_roc = [prob[1] for prob in y_pred_prob]
+
     run_id = run.info.run_id
     params, metrics, tags, artifacts = get_run_data(run_id)
     assert params == truncate_dict(stringify_dict_values(model.get_params(deep=True)))
 
     expected_metrics = {
         TRAINING_SCORE: model.score(X, y_true),
-        _METRICS_PREFIX + "accuracy_score": sklearn.metrics.accuracy_score(y_true, y_pred),
-        _METRICS_PREFIX
+        _TRAINING_PREFIX + "accuracy_score": sklearn.metrics.accuracy_score(y_true, y_pred),
+        _TRAINING_PREFIX
         + "precision_score": sklearn.metrics.precision_score(y_true, y_pred, average="weighted"),
-        _METRICS_PREFIX
+        _TRAINING_PREFIX
         + "recall_score": sklearn.metrics.recall_score(y_true, y_pred, average="weighted"),
-        _METRICS_PREFIX + "f1_score": sklearn.metrics.f1_score(y_true, y_pred, average="weighted"),
-        _METRICS_PREFIX + "log_loss": sklearn.metrics.log_loss(y_true, y_pred_prob),
+        _TRAINING_PREFIX + "f1_score": sklearn.metrics.f1_score(y_true, y_pred, average="weighted"),
+        _TRAINING_PREFIX + "log_loss": sklearn.metrics.log_loss(y_true, y_pred_prob),
```

Comment:
On second thought, I'd rather we use the full strings `training_accuracy_score, `training_f1_score` in our test assertions (it's fine to continue using `_TRAINING_PREFIX` in the actual implementation). Otherwise, someone could come along and change `_TRAINING_PREFIX` to `will_is_awesome`; these tests would still pass, but users would see strange metrics of the form `will_is_awesome_accuracy_score`.